pkgs/by-name/ve/vesktop/package.nix

```
@@ -133,6 +136,20 @@ stdenv.mkDerivation (finalAttrs: {
       runHook postInstall
     '';
 
+  fixupPhase =
+    let
+      libPath = lib.makeLibraryPath [
+        pipewire
+        libpulseaudio
+        stdenv.cc.cc.lib
+      ];
+    in
+    ''
+      find $out/ -name '*.node' -exec patchelf \
+        --add-rpath "${libPath}" \
+        {} \;
+    '';
+
```

we should be able to use `autoPatchelfHook` here. if we do, these dependencies can just go into `buildInputs` and no other change is needed

```suggestion
  buildInputs = [
    pipewire
    libpulseaudio
    stdenv.cc.cc.lib
  ];
```

this change could also be made in other packages to simplify things a bit - though since i'm not familiar with them i can't guarantee it :p

---

Yeah, this works for `vesktop` and `logseq`, but not for `terra-station` and `standardnotes`, where we need to un- and repack the upstream `asar`.